nixos/modules/services/networking/soju.nix

```diff
@@ -66,6 +69,14 @@ in
       description = lib.mdDoc "Whether to enable message logging.";
     };
 
+    adminSocket.enable = mkOption {
```

Comment:
We could use mkEnableOption for this

Reply:
I'm not sure how to do that, because we have `default = true` here.

Reply:
oh, yeah, we could add that but then we don't save much.